Feat/additional path graphic props

Koda-Pig opened the pull request:
## 📝 Description

- Introduced strokeLinecap and strokeLinejoin props to customize stroke appearance.
- Updated component type definitions and Storybook stories to reflect new props.

## 🚨 Notes for Reviewers
N/A

## 📸 Screenshots (if applicable)
N/A

Koda-Pig committed:
feat: add strokeLinecap and strokeLinejoin props to AnimatedPathGraphic component

- Introduced strokeLinecap and strokeLinejoin props to customize stroke appearance.
- Updated component type definitions and Storybook stories to reflect new props.

Koda-Pig committed:
0.0.18

benbennitt commented on src/components/animated/AnimatedPathGraphic.tsx:
```diff
@@ -16,6 +16,8 @@ export default function AnimatedPathGraphic({
   loop = true,
   loopCount = "infinite",
   className = "",
+  strokeLinecap = "butt",
+  strokeLinejoin = "miter",
```
```suggestion
  strokeLinecap = "round",
  strokeLinejoin = "round",
```

benbennitt replied:
Would want round as default, which AFAIK is the native default if not specified, and currently in effect on all Metaversal animated paths.

Koda-Pig replied:
Default value of `stroke-linecap` is `butt` according to [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/stroke-linecap).

Default value of `stroke-linejoin` is `miter` according to [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/stroke-linejoin).


But happy to make those the defaults if you want?

Koda-Pig committed:
refactor: update default values for strokeLinecap and strokeLinejoin props in AnimatedPathGraphic component

- Removed default values for strokeLinecap and strokeLinejoin to allow for more flexible customization.
- Adjusted related styles to ensure consistency with the new prop handling.
